### PR TITLE
Secure URLs (1 of 3)

### PR DESCRIPTION
**Pull request by vszakats**
Ref: https://github.com/Homebrew/homebrew-cask/pull/52975

### CODE REVIEW

**Comment by commitay** [comment]
```
audit for brushviewql: warning
 - SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls
audit for doublecommand: warning
 - SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls
audit for excalibur: warning
 - SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls
```

File: Casks/go2shell.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
   version '2.3'
   sha256 '8cdfbe43a9e0a6728cb2bc5f74bdf0c4087e6d033b829acb68cb5ceaddda44a3'
 
-  url 'http://zipzapmac.com/download/Go2Shell'
+  url 'https://zipzapmac.com/download/Go2Shell'
+  # Alternative without an insecure redirect: 'https://zipzapmac.com/DMGs/Go2Shell.dmg'
```
Comment (commitay) [inline Casks/go2shell.rb]: Lets deal with this in a follow up, we don't really want to add comments in casks.

File: Casks/context-free.rb [hunk 2]
```diff
@@ -2,7 +2,8 @@
   version '3.0.11.3'
   sha256 '5baae1cf7487ea0902f781f0891d3f7dc2c01071ab9546a1e2523676ea45c0da'
 
-  url "http://www.contextfreeart.org/download/ContextFree#{version}.dmg"
+  url "https://www.contextfreeart.org/download/ContextFree#{version}.dmg"
+  # Alternative without an insecure redirect: "https://glyphic.s3.amazonaws.com/cfa/download/ContextFree#{version}.dmg"
```
Comment (commitay) [inline Casks/context-free.rb]: Lets deal with this in a follow up.

File: Casks/cactus.rb [hunk 3]
```diff
@@ -2,10 +2,11 @@
...
   sha256 'af920f5f180b64731adef561a45213064c27830f5d9030d6a862f339ffc4e88c'
 
-  url "http://update.cactusformac.com/static/downloads/Cactus-#{version}.zip"
-  appcast 'http://update.cactusformac.com.s3-website-us-east-1.amazonaws.com/appcast.xml'
+  # update.cactusformac.com was verified as official when first introduced to the cask
+  url "https://update.cactusformac.com/static/downloads/Cactus-#{version}.zip"
+  appcast 'https://s3.amazonaws.com/update.cactusformac.com/appcast.xml'
   name 'Cactus'
-  homepage 'http://cactusformac.com/'
+  homepage 'https://web.archive.org/web/20160801163004/cactusformac.com/'
```
Comment (commitay) [inline Casks/cactus.rb]: Lets deal with this in a follow up, probably need to find a current homepage or delete it.

File: Casks/biba.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@
   url "https://s3-us-west-1.amazonaws.com/downloads.biba.com/builds/Biba-OSX-#{version}.dmg"
   appcast 'https://s3-us-west-1.amazonaws.com/downloads.biba.com/appcast/mac'
   name 'Biba'
-  homepage 'http://www.biba.com/'
+  homepage 'https://web.archive.org/web/20161208050946/www.biba.com/'
```
Comment (commitay) [inline Casks/biba.rb]: Lets deal with this in a follow up, probably need to find a current homepage or delete it.

**Comment by vszakats** [comment]
The three SourceForge failures are also caused by outdated audit rules. `*.sourceforge.io` is the new `*.sourceforge.net`, which also supports HTTPS and doesn't risk sf.net site security due to the distinct domain. It's perfectly legit to store files there for download. (The only caveat is that it is by default redirected to the legacy `*.sourceforge.net` page, unless the project owner manually flipped the HTTPS switch on)

**Comment by commitay** [comment]
Dropping the SourceForge casks that fail audit until the audit is adjusted.

**Review by commitay — APPROVED** [review]

**Pull request merged**